src/highlander/highlander.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Ceros Highlander Plugin. There can be only one (layer visible at a time)
+ * @version 0.2.0
+ * @support support@ceros.com
+ *
+ * This plugin allows you to define a group of layers, where only 1 layer in the group can be visible at a time
+ * If any layer in the group is shown, then all other layers in the group will be automatically hidden. The groups
+ * are defined using the SDK panel to give each layer in the group the same tag.
+ *
+ * You can define multiple groups by giving different tags to each group
+ *
+ * This plugin is licensed under the MIT license. A copy of this license and
+ * the accompanying source code is available at https://github.com/ceros/ceros-plugins
+ */
+
+(function() {
+
+    if (typeof(CerosSDK) === "undefined") {
```

Comment:
It'd be cool if we could have this in a helper and shove it in to each plugin with grunt somehow.